lib/alchemy/upgrader/tasks/ingredients_migrator.rb

```diff
@@ -33,6 +33,7 @@ def create_ingredients
                     else
                       ingredient.value = content.ingredient
                     end
+                    content.destroy!
```

Comment:
Okay, but we should inform people. Currently they have a safety-net. But otherwise they should have a db backup anyway.

But we should also fix the issue in elements_block_helper https://github.com/tvdeyen/alchemy_cms/blob/normalize-contents-and-essences/app/helpers/alchemy/elements_block_helper.rb#L32

The element editor gives precedence to ingredients. We should do in the frontend as well.